.travis.yml

```diff
@@ -0,0 +1,35 @@
+# Travis yml file inspired by scikit-bio
+# Check on http://lint.travis-ci.org/ after modifying it!
+sudo: false
+language: python
+env:
+  - PYVERSION=3.5 USE_CYTHON=TRUE MAKE_DOC=TRUE
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+  - conda update --yes conda
+  - conda info -a
+install:
```

Comment:
Can we make the environment install here congruous with the environment install from README.md?

Reply:
Done, good idea!